x-pack/test_serverless/functional/test_suites/search/search_homepage.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { RoleCredentials } from '../../../shared/services';
+
+import { testHasEmbeddedConsole } from './embedded_console';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['svlCommonPage', 'svlCommonNavigation', 'svlSearchHomePage']);
+  const svlUserManager = getService('svlUserManager');
+  const uiSettings = getService('uiSettings');
+  let roleAuthc: RoleCredentials;
+
+  const HOMEPAGE_FF_UI_SETTING = 'searchHomepage:homepageEnabled';
+  describe('Search Homepage', () => {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
```

Comment:
@dmlemeshko running these locally against MKI I needed to create the `.ftr/role_users.json` file.

Do you know if this will work when running in CI? if not I can tag these to `skipMKI`